components/icu/Cargo.toml

```diff
@@ -45,6 +45,17 @@ icu_segmenter = { version = "1.0.0-alpha1", path = "../../experimental/segmenter
 
 # For docs links
 icu_provider = { version = "1.0.0-beta1", path = "../../provider/core", default-features = false }
+icu_provider_fs = { version = "1.0.0-beta1", path = "../../provider/fs", optional = true }
```

Comment:
I'm wondering if this should be a separate `icu_tutorials` crate instead so that we can be cleaner around dependencies. It can live under experimental/ for now, but eventually I'd like for it to be a source of tutorials on docs.rs